astropy/stats/tests/test_funcs.py

```diff
@@ -9,12 +9,19 @@
 from numpy.testing import assert_equal
 from numpy.testing.utils import assert_allclose
 
+from astropy.utils import minversion
+
 try:
     import scipy  # pylint: disable=W0611
 except ImportError:
     HAS_SCIPY = False
+    SCIPY_LT_1_4 = False
 else:
     HAS_SCIPY = True
+    if minversion(scipy, '0.14.0'):
```

Comment:
I'm confused by all the `not` and `<` and thus it may be that I misunderstood your comment. 

I think a variable should be named by the content not by convention. If it says `LT` it should represent `<` (similar for numpy-variables: https://github.com/astropy/astropy/blob/master/astropy/utils/compat/numpycompat.py#L18). If one wants to stick with `LT` by convention it's easy to put a `not` in front of `minversion` (like for the numpy-variables)

> I would stick with LT even though that means you'd test with not SCIPY_LT_0_14 below.

If the variable contains the LT condition (in name and content) then one doesn't need the `not`.